manim/scene/scene.py

```diff
@@ -863,6 +864,18 @@ def add_static_frames(self, duration):
             num_frames=int(duration / dt),
         )
 
+    def add_click_sound(self, **kwargs):
+        """Add a click sound to the animation.
+
+        .. seealso::
+
+            :meth:`~.add_sound`
+        """
+        path_sound = (
+            Path(__file__).parent.parent.parent / "manim" / "assets" / "click.mp3"
```

Comment:
I still worry that this *may* not work sometimes
```suggestion
            Path(__file__, "../../assets/click.mp3").resolve()
```
^ should be better than using `.parent`